extensions/ecg-dicom/.webpack/webpack.prod.js

```diff
@@ -3,7 +3,7 @@ const pkg = require('../package.json');
 
 const outputFile = 'index.umd.js';
 const rootDir = path.resolve(__dirname, '../');
-const outputFolder = path.join(__dirname, '../dist');
+const outputFolder = path.join(__dirname, '../public/umd/@radical/ecg-dicom/');
```

Comment:
Naming convention needs to be such that the distribution can be served up by the default OHIF build, and can be found just by knowing the names here.